Documentation/contributing/development/contributing_guide.rst

```diff
@@ -14,12 +14,14 @@ Clone and Provision Environment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 #. Make sure you have a `GitHub account <https://github.com/join>`_
-#. Clone the cilium repository
+#. Clone the cilium repository into your ``GOPATH``.
 
    ::
 
-      go get -d github.com/cilium/cilium
-      cd $GOPATH/src/github.com/cilium/cilium
+      mkdir -p $GOPATH/src/github.com/cilium
```

Comment:
With modules this doesn't need to be the case, I just have the repo cloned at `~/Code/cilium` and `GOPATH=~/Code/cilium/.gopath`, it works perfectly for me. I don't insist that we document my setup as such, but just saying that `src/github.com/cilium` prefix and pre-existing `GOPATH` are not essential and there is a lot of flexibility.

Reply:
@errordeveloper @Rolinh @tklauser @joestringer in an ideal world that is correct but we use kubernetes scripts that depend a lot on GOPATH. Each developer is free to set the project on the directory that suits better his / her dev workflow but for new corners is better to have these instructions so they don't have any issues if they try to use the k8s scripts. (`make generate-k8s-api`)

Reply:
I see, thanks for clarifying.

Reply:
https://github.com/cilium/cilium/issues/11341